Formula/pandoc-crossref.rb

```diff
@@ -1,8 +1,8 @@
 class PandocCrossref < Formula
   desc "Pandoc filter for numbering and cross-referencing"
   homepage "https://github.com/lierdakil/pandoc-crossref"
-  url "https://hackage.haskell.org/package/pandoc-crossref-0.3.15.0/pandoc-crossref-0.3.15.0.tar.gz"
-  sha256 "b9d8b6ed246fcac6955e14d38b229b62b0682b98ec2caf7fdc095438f16dfd4f"
+  url "https://github.com/lierdakil/pandoc-crossref/archive/v0.3.15.0.tar.gz"
+  sha256 "d0780e08bae9b589eaad01be1fc818ab551f7debe26a9d1be57aed58032a6b5e"
   license "GPL-2.0-or-later"
```

Comment:
```suggestion
  license "GPL-2.0-or-later"
  revision 1
```

Reply:
Odd that I can't merge my own suggestion onto this branch…

Reply:
Oh, looks like the perms got adjusted and I can't make changes on this branch now. @SMillerDev Can you merge that suggestion?